### PR TITLE
Added my profile

### PR DESCRIPTION
**Pull request by mrmorris**

### CODE REVIEW

**Comment by mrmorris** [comment]
Resolves issue #8 check it out @reidaelliott

File: ryan-morris.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<h1>Ryan</h1>
+<ul>
+<li>Ski</li>
+<li>eradicating greasy molues</li>
```
Comment (reidaelliott) [inline ryan-morris.html]: please add one more

Comment (reidaelliott) [inline ryan-morris.html]: looks good

**Comment by reidaelliott** [comment]
looks good

**Pull request merged**